src/general/block-endpoint-edit-role/src.rego

```diff
@@ -0,0 +1,32 @@
+package blockendpointeditrole
+
+violation[{"msg": msg}] {
+    input.review.object.metadata.name == "system:aggregate-to-edit"
+    endpointRule(input.review.object.rules[_])
+    msg := "ClusterRole system:aggregate-to-edit should not allowed endpoint permissions"
+}
+
+violation[{"msg": msg}] {
+    input.review.object.metadata.name == "system:aggregate-to-edit"
+    not disableAutoupdate(input.review.object.metadata.annotations)
+    msg := "ClusterRole system:aggregate-to-edit is required rbac.authorization.kubernetes.io/autoupdate=false after reconciled none endpoint permission"
```

Comment:
nit: s/ClusterRole system:aggregate-to-edit is required/ClusterRole system:aggregate-to-edit requires

Reply:
what does `after reconciled none endpoint permission` mean here?

Reply:
thx and reword this into another violation msg